Manifests/ManagedPreferencesApplications/de.fau.rrze.NetworkShareMounter.plist

```diff
@@ -149,16 +209,72 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_default</key>
 			<true/>
 			<key>pfm_description</key>
-			<string>Configure a help URL to help users interact with the application.</string>
+			<string>Show exit button in the menu bar.</string>
 			<key>pfm_name</key>
-			<string>unmountOnExit</string>
+			<string>canQuit</string>
 			<key>pfm_title</key>
-			<string>Unmount Shares on Exit</string>
+			<string>Show Exit</string>
 			<key>pfm_type</key>
 			<string>boolean</string>
 			<key>pfm_app_min</key>
 			<string>2</string>
 		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string></string>
+			<key>pfm_description</key>
+			<string>Kerberos/AD Domain for user authentication. If set, automatic AD/Kerberos authentication and ticket renewal will be enabled.</string>
+			<key>pfm_name</key>
+			<string>kerberosRealm</string>
+			<key>pfm_title</key>
+			<string>Kerberos Realm</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_app_min</key>
+			<string>2</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
```

Comment:
Are you sure that these new subkeys `keychainiCloudSync`, `keyChainService`, and `keyChainComment` were already available in version 2.0.0?

Reply:
I couldn't find information about those keys in the README so I removed them.